test/math.jl

```diff
@@ -952,3 +952,24 @@ float(x::FloatWrapper) = x
     @test isa(sin(z), Complex)
     @test isa(cos(z), Complex)
 end
+
+module HypotTest
+    import Base: abs, isless, isinf, oneunit, /, *
+    using Test
+
+    struct MeterUnits{T} <: Number
```

Comment:
Can https://github.com/JuliaLang/julia/blob/master/test/dimensionful.jl be used here instead?

Reply:
Huh, I didn't realize that existed. It would seem that at various places in the test suite, `MeterUnits`, `PhysQuantity`, and `Furlong` are defined. A separate PR could possibly consolidate all that.

I've followed what was done [here](https://github.com/JuliaLang/julia/blob/686928f830b9ad962903b56982209784cbf330c3/test/ranges.jl#L1218) to use TestHelpers in test/math.jl. I hope that's correct.